get_images.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
+
+"""
+    images.py
```

Comment:
You could name this as "get_page_images.py"

Reply:
Changed name on file and in comment with latest commit